manifests/macvlan-cni-config/macvlan-net-cr.yml

```diff
@@ -0,0 +1,23 @@
+apiVersion: "k8s.cni.cncf.io/v1"
```

Comment:
can you rename the folder to:

`stage-macvlan-network`

for uniformity with other folders under manifest

also lets name it in the same format as the others: `0010-macvlan-net-cr.yml`